content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -102,13 +102,21 @@ to ensure that the API presents a clear, consistent view of system resources
 and behavior, and to enable controlling access to end-of-life and/or
 experimental APIs.
 
-Refer to [API versions reference](/docs/reference/using-api/api-overview/#api-versioning)
-for more details on the API version level definitions.
-
 To make it easier to evolve and to extend its API, Kubernetes implements
 [API groups](/docs/reference/using-api/api-overview/#api-groups) that can be
 [enabled or disabled](/docs/reference/using-api/api-overview/#enabling-or-disabling).
 
+API resources are distinguished from each other based on their API groups and
```

Comment:
Here is a suggested edit:
API resources are distinguished by their API group, version, and resource type
known as their `kind`. The API server may serve more than one API version for the same resource
and attempt to handle the conversion between API versions transparently.
For example, suppose there are two versions `v1` and `v1beta1` for the same resource.
An object created by the `v1beta1` version can then be read, updated, and deleted by the `v1` version as long as the
identifying fields for that resource are provided.

Reply:
Done.

Reply:
> API resources are distinguished by their API group, version, and resource type known as their kind.

Note that the above statement is incorrect. `version` does NOT help distinguish API resources and that is exactly what we are trying to clarify here.

Reply:
OK. I was under the impression that Group, Version, Kind uniquely represent (when combined) a
resource.